lms/static/sass/base/_grid-settings.scss

```diff
@@ -7,7 +7,7 @@ $border-box-sizing: false;
 
 
 /* Breakpoints */
-$mobile: new-breakpoint(max-width 320px 4);
-$tablet: new-breakpoint(min-width 321px max-width 768px, 8);
+$mobile: new-breakpoint(max-width 540px 4);
```

Comment:
@marjev - the example that you give above still includes a breakpoint at 320px. I really think we must add a new breakpoint if necessary instead of changing an existing one.